webadmin/urls.py

```diff
@@ -14,4 +15,6 @@
     url(r'^admin/restore_users', views.restore_user, name='admin/restore_users'),
     url(r'^admin/delete_user', views.delete_user, name='admin/delete_user'),
     url(r'^admin/adduser/$', views.add_user, name='admin/adduser'),
```

Comment:
@madhumithanara weren't this urls changed to `path` in the previous PR?
Please do so, so that we have consistency in the code usage